recipes/configs/generation.yaml

```diff
@@ -27,11 +27,10 @@ tokenizer:
   _component_: torchtune.models.llama2.llama2_tokenizer
   path: /tmp/Llama-2-7b-hf/tokenizer.model
   max_seq_len: null
+  prompt_template: null
 
 # Generation arguments; defaults taken from gpt-fast
 prompt: "Tell me a joke?"
```

Comment:
I'm wondering if we should just adopt a format like generation_v2 here where we have the default be to make it clear:

```yaml
prompt:
	user: Tell me a joke.
```

Reply:
yeah that works. was just trying to make the minimal changes needed since we are migrating to generate_c2